Sources/BraintreeCore/Analytics/FPTIBatchData.swift

```diff
@@ -110,6 +113,7 @@ struct FPTIBatchData: Codable {
             case requestStartTime = "request_start_time"
             case timestamp = "t"
             case tenantName = "tenant_name"
+            case sessionID = "session_id"
```

Comment:
I don't think we should need a new property for this since a "session_id" tag already exists in the `Metadata` - we should be able to pass the session ID from `ShopperInsightsClient` to that existing field instead. @jaxdesmarais correct me if I'm misunderstanding the requirements

Reply:
You're correct, we should be setting the existing session ID in Metadata as this value vs creating a new property.

Reply:
thanks, don't know how I didn't see `session_id` in the metadata at all, otherwise I wouldn't have added it to events